src/components/AudioPlayer/index.js

```diff
@@ -198,6 +198,16 @@ class AudioPlayer extends Component {
       keepPlaying,
     } = this.props
     const { width, track } = this.state
+
+    // If the track URL isn't available, the audio player will crash native apps.
+    // Display a simple message notifying the user of the problem, and prevent the
+    // player from loading and crashing the app.
+    // The URL is usually unavailable in the editor, which doesn't cause any harm, so
+    // allow the audio player to load in that context.
+    if (!editor && !track?.url) {
+      return <Text>Unable to load audio player, audio track URL unavailable.</Text>
+    }
+
```

Comment:
So will Users see this error every time the Audio Player initially loads while we request audio urls?

Reply:
From my experience, they won't.  But if they did, it would only be for a very brief period of time.  
Keep in mind this is more of a quick fix, and we ought to implement better handling of asynchronous data handling and component loading.

Reply:
I'm seeing this quite often. We should have a loading state instead.